redis/commands/bf/commands.py

```diff
@@ -233,6 +234,15 @@ def exists(self, key, item):
         params = [key, item]
         return self.execute_command(CF_EXISTS, *params)
 
+    def mexists(self, key, *items):
+        """
+        Check whether an `items` exist in Cuckoo Filter `key`.
+        For more information see `CF.MEXISTS <https://redis.io/commands/cf.mexists>`_.
+        """  # noqa
+        params = [key]
+        params += items
+        return self.execute_command(CF_MEXISTS, *params)
```

Comment:
Why do you need the params? WDYT about:
```suggestion
        return self.execute_command(CF_MEXISTS, key, *items)
```